$OpenBSD: patch-scene_gui_rich_text_label_cpp,v 1.1.1.1 2018/04/27 07:32:53 bentley Exp $

fix the invalid/undefined assignment to nullptr of int type (clang6?)

Index: scene/gui/rich_text_label.cpp
--- scene/gui/rich_text_label.cpp.orig
+++ scene/gui/rich_text_label.cpp
@@ -829,9 +829,9 @@ void RichTextLabel::_gui_input(Ref<InputEvent> p_event
 							// Erase previous selection.
 							if (selection.active) {
 								selection.from = NULL;
-								selection.from_char = NULL;
+								selection.from_char = '\0';
 								selection.to = NULL;
-								selection.to_char = NULL;
+								selection.to_char = '\0';
 								selection.active = false;
 
 								update();
